MapboxNavigation/NavigationMapView.swift

```diff
@@ -329,7 +332,7 @@ open class NavigationMapView: MGLMapView, UIGestureRecognizerDelegate {
         tracksUserCourse = false
     }
     
-    @objc public func updateCourseTracking(location: CLLocation?, animated: Bool) {
+    @objc func updateCourseTracking(location: CLLocation?, animated: Bool) {
```

Comment:
Should we leave this method public? Is there still a use case for manually updating the course view independently of a route controller?

Reply:
🤔maybe. I think it might be wise to wrap these types of functions in a more specific helper function like recenterMap:

https://github.com/mapbox/mapbox-navigation-ios/blob/0b69c3247f500c579bfa1b97f95d5f07f844348d/MapboxNavigation/NavigationMapView.swift#L1064-L1067